### PR TITLE
Improve imagenet example performance

### PR DESCRIPTION
**Pull request by okuta**
The dtype of `mean` is `numpy.float32`.
Division operation is slow.

### CODE REVIEW

File: examples/imagenet/train_imagenet.py [hunk 1]
```diff
@@ -61,7 +61,7 @@ def get_example(self, i):
 
         image = image[:, top:bottom, left:right]
         image -= self.mean[:, top:bottom, left:right]
-        image /= 255
+        image *= (1.0 / 255.0)
```
Comment (zori) [inline examples/imagenet/train_imagenet.py]: Unrelated to  speed-up, could you add a comment above `# Scale to [0, 1]`. Although trivial, the other pre-processing steps are documented.

Comment (okuta) [inline examples/imagenet/train_imagenet.py]: OK.

**Comment by zori** [comment]
LGTM! #2000 :)

**Comment by okuta** [comment]
Thank you!

**Pull request merged**